hw1/02-palindrome.html

```diff
@@ -6,22 +6,32 @@
     <title>02 - Palindrome</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="02-palindrome.js"></script>
   </head>
-  <body>
-    <main>
+  <body style="background-color: blue;">
+    <main class="m-5">
       <section>
+        <form class="form bg-light border rounded mx-auto p-2" action="" 
+        style="width:500px;">
         <h1>Exercise 02 - Palindrome</h1>
-        <div>
-          <label>Enter a positive number:</label>
-          <input type="number" />
+        <div class="form-group-mb4">
+          <div class ="row" >
+            <div class="col-md-3">
+              <label for="enternumber" style="width: 300px;">Enter a positive number:</label>
+            </div>
+            <div class="col-md-9">
+              <input  type="number" name="enternumber" class="form-control" style="width: 300px;margin-left: 50px;"/>
```

Comment:
I'm getting a WAVE error from this input control not being associated with a label. I think you are trying to use the name attribute to link the label to this input, but I think you need to use an 'id' attribute instead for it to work.

Reply:
Thanks for catching that! I have fixed it.